packages/widgets/README.md

```diff
@@ -23,12 +23,27 @@ npm install @latticejs/widgets
 import React, { Component } from 'react';
 import { Loader } from '@latticejs/widgets';
 
-const App = () => {
```

Comment:
Kindly use hooks inside the ReadMe instead of components. Kindly revert the ReadMe.

Reply:
Ok